src/Foundation.Find.Cms/Config/EnumSelectionDescriptionAttribute.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.ComponentModel;
+
+namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done